test/test_yaml.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from unstructured_platform_plugins.cli.schema_yaml.yaml_generator import generate_yaml
+
+test_path = Path(__file__).parent
+results_path = test_path / "expected_results"
+
+
+def test_generate_yaml_simple():
+    output = generate_yaml(app="test.assets.typed_dict_response:sample_function")
+    file_path = results_path / "simple.yaml"
```

Comment:
Nitpick: there is no obvious relation between `sample_function` stored in `test.assets.typed_dict_response.py`, and `expected_results/simple.yaml` . Naming it like `expected_results/typed_dict_response:sample_function` would make it painfully clear, unfortunately it's a mouthful. Maybe it's ok, or maybe just using the filename, or a function name, would be good enough. I'll leave it for you to decide, to not keep this PR waiting any longer.